Statsdash/GA/aggregate_data.py

```diff
@@ -146,8 +158,11 @@ def article_table(self):
                 rows = utils.change_key_names(rows, {"title":"pageTitle", "path":"pagePath", "host":"hostname"})
                 for row in rows:
                     path = row["path"]
+                    title = row["title"]
                     new_path = self._remove_query_string(path)
+                    new_title = self._check_for_amp(path, title)
```

Comment:
`_check_for_amp` is a misnomer for what the helper method does making reading the code a little less self-documenting.  What about making the helper a bit more generic like `self._get_title(row)`?